src/config.ts

```diff
@@ -197,6 +196,7 @@ export const Config = Schema.intersect([
     Schema.object({
       type: Schema.const('sd-webui'),
       sampler: sampler.createSchema(sampler.sd),
+      upscaler: Schema.union(sdUpscalers.map((up) => Schema.const(up))).description('默认的放大算法。').default('Lanczos'),
```

Comment:
```suggestion
      upscaler: Schema.union(upscalers).description('默认的放大算法。').default('Lanczos'),
```

Array of primitives will be cast into const.

Reply:
That's good.